fluent-syntax/src/serializer.js

```diff
@@ -40,10 +40,12 @@ export default class FluentSerializer {
         return serializeMessage(entry);
       case 'Section':
         return serializeSection(entry);
-      case 'Comment': {
-        const comment = serializeComment(entry);
-        return `\n${comment}\n\n`;
-      }
+      case 'Comment':
+        return `\n${serializeComment(entry)}\n\n`;
```

Comment:
I think we should have a flag on the `FluentSerializer` class which starts as `isFirstEntry = true` and only add those leading `\n` if it's `false`.